packages/react-admin/src/mui/layout/Layout.js

```diff
@@ -83,7 +83,7 @@ const Layout = ({
     <MuiThemeProvider theme={createMuiTheme(theme)}>
         <div className={classnames(classes.root, className)} {...rest}>
             <div className={classes.appFrame}>
-                <Hidden xsDown>
+                <Hidden smDown>
```

Comment:
needs more thorough tests ; unless something changed on the mui side, this looks odd.

Reply:
Something changed. It's not inclusive anymore: https://github.com/mui-org/material-ui/releases/tag/v1.0.0-beta.24